src/popup/router/components/AmountSend.vue

```diff
@@ -23,7 +23,7 @@
       <div class="balance-box">
         <p class="label">{{ $t('pages.tipPage.availableLabel') }}</p>
         <span class="secondary-text f-14 block l-1" data-cy="balance">
-          {{ selectedToken ? selectedToken.convertedBalance : tokenBalance }}
+          {{ selectedToken ? selectedToken.convertedBalance : tokenBalance.toFixed(2) }}
```

Comment:
Why `.toFixed(2)`?

The fungible tokens can have as many decimal pieces as they want to. 🤔

Reply:
https://github.com/aeternity/superhero-wallet/pull/652/files#diff-42d961a783c297603d8cd3d38fb7471e62b81eb03a5ac50e7cb3124042e4b495L44-L48

Reply:
Ok. Then we need to discuss this functionality, but not in the scope of this PR.